app/scripts/controllers/metametrics-controller.ts

```diff
@@ -175,6 +176,7 @@ export type MetaMaskState = {
     };
   };
   ///: END:ONLY_INCLUDE_IF
+  keyrings: { type: string; accounts: string[] }[];
```

Comment:
Is the `accounts` property used?

Reply:
Not in this PR, but `accounts` are there, we're just adding something that was passed in the state anyway, but was not added to the type.

Reply:
👍